OpenRA.Utility/Program.cs

```diff
@@ -45,7 +45,13 @@ static void Main(string[] args)
 
 			Game.InitializeSettings(Arguments.Empty);
 
-			var modSearchPaths = new[] { Path.Combine(".", "mods"), Path.Combine("^", "mods") };
+			var envModSearchPaths = Environment.GetEnvironmentVariable("MOD_SEARCH_PATHS");
+			var modSearchPaths = !string.IsNullOrWhiteSpace(envModSearchPaths) ? envModSearchPaths.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries)
+				: new[] { Path.Combine(".", "mods"), Path.Combine("^", "mods") };
+
+			for (var i = 0; i < modSearchPaths.Length; i++)
```

Comment:
It would be cleaner to use
```csharp
envModSearchPaths.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries)
	.Select(x => x.Trim())
	.ToArray()
```

and then drop the explicit loop.